### ISSUE TITLE
npm test fails and I cannot start the server

### ISSUE THREAD

**Issue by aniketkk** [linnovate/mean#1554]
<img width="1440" alt="screen shot 2016-05-03 at 1 23 08 am" src="https://cloud.githubusercontent.com/assets/7087620/14975240/b67ad79a-10cd-11e6-8ccd-c98ccefe8632.png">

server does not start either - i cant load localhost:3000
<img width="1440" alt="screen shot 2016-05-03 at 1 25 01 am" src="https://cloud.githubusercontent.com/assets/7087620/14975275/fcf8b8c2-10cd-11e6-9d44-3b861a18d398.png">

Please help.
osX el capitan 10.11.4
i have run all the commands without sudo. 
Please help! 
this is probably the same issue as this .#1532

**Comment by kajo404** [comment]
having the same issue

**Comment by timelf123** [comment]
@kajo404 are you also on mac? the only time I have seen this bug has been on my co-workers mac machines

**Comment by zeeshanbashir** [comment]
having same issue :(

**Comment by timelf123** [comment]
@zeeshanbashir please share the requested details in https://github.com/tinymce/tinymce/issues/2836

**Comment by aniketkk** [comment]
@timelf123  assuming this problem exists for the same config can you post a solution please. If you do know the solution that is.

**Comment by timelf123** [comment]
Please share your:
OS
node -v
npm -v
mean version
how did you install mean? cloning this repo or CLI install?

**Comment by aniketkk** [comment]
node - v v4.4.3
npm -v 3.8.8

**Comment by igorg1312** [comment]
Whatz test fails, run the test From shell?
Il 04 Mag 2016 06:08, "aniketkk" <notifications@github.com> ha scritto:

**Comment by twistymcgee** [comment]
I am also running into this issue. Everything is fresh install, including the OS since it's only 2 weeks old:

OS X El Capitan 10.11.4
Node: 4.4.3
Npm: 2.15.1

**Comment by pashist** [comment]
#1560

**Comment by pedramp** [comment]
same issue.
OSX 10.11

$ node --version
v4.4.4
$ mean --version
0.12.1
$ npm --version
2.15.1

**Comment by dnithinraj** [comment]
Same issue. localhost:3000 is not loading - please help.

**Comment by timelf123** [comment]
Works fine for me, make sure all of your dependencies are up to date. If in doubt, reclone this repo.


```
➜ npm -v
3.7.3
➜ node -v
v5.9.1
➜ mean status

    MEAN Status
    -----------

    MEAN VERSION: 0.6.1

    DB connection successful!

    MongoDB URI: ...

    checking meanio and global mean-cli versions
    meanio at latest version: 0.9.1
    mean-cli at latest version: 0.11.1
➜ npm test

> mean@0.6.1 test /home/tim/dev/crowds-prediction-market
> gulp test

Invoking gulp - development
[10:51:50] Using gulpfile ~/dev/crowds-prediction-market/gulpfile.js
[10:51:50] Starting 'startServer'...
================================SOCKETS SERVER STARTED===========================
Mean app started on port 3001 (test) cluster.worker.id: 0
[10:51:53] Finished 'startServer' after 2.87 s
[10:51:53] Starting 'runMocha'...


  0 passing (1ms)

[10:51:55] Finished 'runMocha' after 1.87 s
[10:51:55] Starting 'runKarma'...
06 06 2016 10:51:55.197:WARN [config]: "/" is proxied, you should probably change urlRoot to avoid conflicts
06 06 2016 10:52:12.126:INFO [karma]: Karma v0.13.22 server started at http://localhost:9876/
06 06 2016 10:52:12.159:INFO [launcher]: Starting browser PhantomJS
06 06 2016 10:52:26.744:INFO [PhantomJS 2.1.1 (Linux 0.0.0)]: Connected on socket /#uJPGmkLDc3bNlUp8AAAA with id 79629614
::ffff:127.0.0.1 - - [06/Jun/2016:15:52:41 +0000] "GET /_getModules HTTP/1.1" 200 999
::ffff:127.0.0.1 - - [06/Jun/2016:15:52:43 +0000] "GET /api/get-public-config HTTP/1.1" 200 154
::ffff:127.0.0.1 - - [06/Jun/2016:15:52:43 +0000] "GET /api/get-public-config HTTP/1.1" 200 154
PhantomJS 2.1.1 (Linux 0.0.0): Executed 1 of 1 SUCCESS (0.02 secs / 1.421 secs)
[10:52:44] Finished 'runKarma' after 50 s
[10:52:44] Starting 'stopServer'...
```

**Comment by dnithinraj** [comment]
Hi,

Please see below snapshots:

[image: Inline images 1]

[image: Inline images 2]

I am not able to update npm nor able to run mean status. I am new to this,
so please advise next steps.

Thanks,
Nithin

**Comment by diptenagile** [comment]
Did anyone fix this issue ? b'coz getting same error.

**Comment by diptenagile** [comment]
Got fixed with node version 6.2.2
